src/main/kotlin/study/racingcar/service/RacingCarGame.kt

```diff
@@ -0,0 +1,28 @@
+package study.racingcar.service
+
+import study.racingcar.domain.Car
+import study.racingcar.domain.RaceResult
+import study.racingcar.util.RacingCarRandomGenerator
+
+class RacingCarGame(
+    private val carCount: Int,
+    private val tryCount: Int,
+    private val cars: List<Car> = (1..carCount).map { Car() },
+    private var raceResults: List<RaceResult> = ArrayList()
+) {
+    fun start() {
+        val initialResults = mutableListOf<RaceResult>()
+        for (i in 1..this.tryCount) {
+            initialResults.add(race(this.cars))
+        }
+
+        this.raceResults = initialResults.toList()
+    }
```

Comment:
```suggestion
      fun start() {
        this.raceResults = (0 until tryCount).map {
            race(cars)
        }
    }
```
이렇게도 구현이 가능할 것 같아요 :)